polaris-tokens/src/utilities.ts

```diff
@@ -118,7 +125,19 @@ export function getCustomPropertyNames(tokens: Tokens) {
     .flat();
 }
 
-export type BreakpointsTokenGroup = typeof breakpointsTokenGroup;
+export function removeMetadata<T extends Exact<MetaTokenGroup, T>>(
+  tokenGroup: T,
+): ExtractValues<T> {
+  return Object.fromEntries(
+    Object.entries(tokenGroup).map(([tokenName, {value}]) => [
+      tokenName,
+      value,
+    ]),
+  ) as ExtractValues<T>;
+}
```

Comment:
```suggestion
export function removeMetadata<T extends Exact<MetaTokenGroup, T>>(
  tokenGroup: T,
) {
  return Object.fromEntries(
    Object.entries(tokenGroup).map((entry): Entry<TokenGroup> => {
      const [tokenName, {value}] = entry as Entry<MetaTokenGroup>;

      return [tokenName, value];
    }),
  ) as ExtractValues<T>;
}
```

Light suggestion to cast the `entry` itself in addition to the final `Object.fromEntries` result. This way we can validate each `.map` return type matches `Entry<TokenGroup>`.